webpack.config.js

```diff
@@ -46,5 +44,17 @@ const plugins = config.plugins.filter((plugin) => !(plugin instanceof ModifyEntr
 
 module.exports = {
   ...config,
-  plugins
+  plugins,
+  output: {
+    uniqueName: 'onecx-announcement-ui',
+    publicPath: 'auto'
+  },
+  experiments: {
+    ...config.experiments,
+    topLevelAwait: true
+  },
+  optimization: {
+    runtimeChunk: false,
+    splitChunks: false
+  }
```

Comment:
needed?

Reply:
topLevelAwait -> needed to transform module into async module
runtimeChunk -> ?
splitChunks -> ?